pkg/logql/syntax/lex.go

```diff
@@ -215,7 +215,9 @@ func (l *lexer) Lex(lval *exprSymType) int {
 	}
 
 	tokenText := l.TokenText()
-	tokenNext := tokenText + string(l.Peek())
+	tokenTextLower := strings.ToLower(l.TokenText())
+	tokenNext := strings.ToLower(tokenText + string(l.Peek()))
```

Comment:
i don't know why we peek here for functions, next token might also need to be covered I think

Reply:
not sure either, I was also wondering about it.. I just added the toLower func to the existing implementation